src/Shell/ServerShell.php

```diff
@@ -69,6 +76,7 @@ public function initialize()
         $this->_host = self::DEFAULT_HOST;
         $this->_port = self::DEFAULT_PORT;
         $this->_documentRoot = WWW_ROOT;
+        $this->_iniPath = '';
```

Comment:
This seems unnecessary, why not just set it to empty string when declaring the property?

Reply:
Followed the pattern for everything else. I can"fix" in a follow-up or.